Watchman.Discord/Areas/Initialization/Services/ServerScanningService.cs

```diff
@@ -20,58 +22,75 @@ public ServerScanningService(ICommandBus commandBus, MessagesHistoryService mess
             _messagesHistoryService = messagesHistoryService;
         }
 
-        public async Task ScanChannelHistory(DiscordServerContext server, ChannelContext channel)
+        public async Task ScanChannelHistory(DiscordServerContext server, ChannelContext channel, DateTime? startTime = null) // startTime ->->-> now
         {
-            const int LIMIT = 100000;
+            const int LIMIT = 20000; // low limit to use props of startTime optimization
```

Comment:
fajnie gdyby to szło z konfiguracji
nie musi to być zrobione teraz, ale dobrze by było, gdyby dało sie tym zarządzać z jednego miejsca, a nie za każdym razem zmieniać kod 

podobnie antispam (ale to chyba już do innego taska)